src/components/SingleDatePicker.jsx

```diff
@@ -242,6 +250,13 @@ class SingleDatePicker extends React.PureComponent {
     });
   }
 
+  onFocusOut(e) {
+    if (!this.container.contains(e.relatedTarget)) {
```

Comment:
hmm, looks like we don't document that we rely on `.contains` on HTML elements, but we use this elsewhere, so it's not a new requirement.

Reply:
Not sure that we need this, and I will investigate if there's another approach - but it seems that we should document that regardless if other things are using it. Is the documentation in the README, elsewhere, or both?

Reply:
It should go in the readme; it's not currently there.